instrumentation/opentelemetry-instrumentation-fastapi/src/opentelemetry/instrumentation/fastapi/__init__.py

```diff
@@ -285,10 +286,16 @@ def uninstrument_app(app: fastapi.FastAPI):
         app._is_instrumented_by_opentelemetry = False
 
     def instrumentation_dependencies(self) -> Collection[str]:
+        # need to use get_distribution because find_spec("fastapi") will return
+        # something even with just fastapi-slim installed
+        try:
+            get_distribution("fastapi-slim")
```

Comment:
Wouldn't it be more reliable to use get_distribution for both?

Reply:
I don't know, but using it we can drop an import